### PR TITLE
docs: expose documentation for randomizers

### PR DESCRIPTION
**Pull request by ST-DDT**
Currently, we sometimes mention the `generateMersenne32Randomizer` and `generateMersenne53Randomizer` methods, but we don't have actual documentation for them on our website.

This PR adds both methods to the Utilities category.

Documentation:

- https://deploy-preview-3221.fakerjs.dev/api/utils.html#generatemersenne32randomizer
- https://deploy-preview-3221.fakerjs.dev/api/utils.html#generatemersenne53randomizer

### CODE REVIEW

File: scripts/apidocs/processing/class.ts [hunk 1]
```diff
@@ -192,7 +192,12 @@ export function processProjectUtilities(project: Project): RawApiDocsPage {
     deprecated: undefined,
     description: 'A list of all the utilities available in Faker.js.',
     examples: [],
-    methods: processProjectFunctions(project, 'mergeLocales'),
+    methods: processProjectFunctions(
+      project,
+      'mergeLocales',
+      'generateMersenne32Randomizer',
+      'generateMersenne53Randomizer'
+    ),
```
Comment (xDivisionByZerox) [inline scripts/apidocs/processing/class.ts:200]: We corrected the project structure in #3141. Thats means that the `src/utils` directory holds all the utilities we provide. Should we automate the feature detection here? That way we will not forget exposing documentation for utilities again, in the future.

Comment (ST-DDT) [reply scripts/apidocs/processing/class.ts:200]: I'll look into that.

Comment (xDivisionByZerox) [reply scripts/apidocs/processing/class.ts:200]: Doesn't nee to be in this PR. We can get this done and look into it in the future. Just wanted to know your thoughts about this.

**Review by xDivisionByZerox — APPROVED** [review]

File: scripts/apidocs/processing/class.ts (hunk 1 above)
Comment (ST-DDT) [reply scripts/apidocs/processing/class.ts:200]: Done

- #3242

**Comment by ST-DDT** [comment]
Merging earlier for inclusion in release.

**Pull request merged**